### DISCUSSION TITLE
error: unknown option `--watch-admin'

### DISCUSSION THREAD

**Discussion by soroka41** [strapi/strapi#5821] [Legacy Topics]
error: unknown option `--watch-admin'

**Comment by soroka41** [comment]
node v10.16.1
npm v6.9.0
OS Windows 10

package
{
  "name": "cms-strapi",
  "private": true,
  "version": "0.1.0",
  "description": "A Strapi application",
  "scripts": {
    "develop": "strapi develop",
    "start": "strapi start",
    "build": "strapi build",
    "strapi": "strapi"
  },
  "devDependencies": {},
  "dependencies": {
    "strapi": "3.0.0-beta.19.5",
    "strapi-admin": "3.0.0-beta.19.5",
    "strapi-utils": "3.0.0-beta.19.5",
    "strapi-plugin-content-type-builder": "3.0.0-beta.19.5",
    "strapi-plugin-content-manager": "3.0.0-beta.19.5",
    "strapi-plugin-users-permissions": "3.0.0-beta.19.5",
    "strapi-plugin-email": "3.0.0-beta.19.5",
    "strapi-plugin-upload": "3.0.0-beta.19.5",
    "strapi-connector-bookshelf": "3.0.0-beta.19.5",
    "knex": "<0.20.0",
    "mysql": "latest"
  },
  "author": {
    "name": "A Strapi developer"
  },
  "strapi": {
    "uuid": "f2f7ef90-4bed-4715-88f8-c27ea15415f8"
  },
  "engines": {
    "node": ">=10.0.0",
    "npm": ">=6.0.0"
  },
  "license": "MIT"
}

**Comment by lauriejim** [comment]
Can you please detail your steps and error you have.
With current info, I will not be able to help you.

**Reply by admhemed** [reply]
just install strapi and try to run the app with strapi develop --watch-admin
https://strapi.io/documentation/3.0.0-beta.x/admin-panel/customization.html#development-mode

**Comment by gdnthomas** [comment]
Same problem here on vagrant & virtual box with a shared folder. This bug is similar to some we encounter with shared files between window and vagrant. The watch does not seems to know the files.
